pkg/remotewrite/config.go

```diff
@@ -243,18 +201,14 @@ func GetConsolidatedConfig(jsonRawConf json.RawMessage, env map[string]string, a
 	}
 
 	// envconfig is not processing some undefined vars (at least duration) so apply them manually
-	if flushPeriod, flushPeriodDefined := env["K6_PROMETHEUS_FLUSH_PERIOD"]; flushPeriodDefined {
-		if err := result.FlushPeriod.UnmarshalText([]byte(flushPeriod)); err != nil {
+	if flushInterval, flushIntervalDefined := env["K6_PROMETHEUS_PUSH_INTERVAL"]; flushIntervalDefined {
```

Comment:
It seems like the varial still has a `flush` in the name, but it's not super-critical

Reply:
Yeah, I confused the two names :facepalm: